keyboards/download.php

```diff
@@ -11,7 +11,7 @@
   //   mode:     optional, either bundle or standalone (for now, supported only for Windows)
   //   cid:      optional, adds client id for Google Analytics tracking
```

Comment:
Aren't the four parameters required now? (in the order `$cid, $id, $platform, $mode`)

Reply:
nvm, I see the default values in `validateParameters()`